saleor/checkout/fetch.py

```diff
@@ -342,7 +356,10 @@ def _apply_voucher_on_product(
         if cheapest_line:
             discounted_lines_by_voucher.append(cheapest_line)
     else:
-        discounted_lines_by_voucher.extend(get_discounted_lines(lines_info, voucher))
+        voucher_info = fetch_voucher_info(voucher)
```

Comment:
question: Why are we fetching voucher info if we already have it? `voucher_info: "VoucherInfo",` Maybe it's just a naming issue?